eco-map/web-app
# Start frontend part with new Preact

**Summary**
Start frontend part with SPA based on new `Preact` v10

**Changed**
1) Created `frontend` folder
2) Installed `preact` and `parcel` for building app
3) Created test-app with one page (use `npm start` or `npm run build` for checking)

**Test-plan**
1) Pull changes
2) Inside `frontend` folder run `npm i`
3) Run `npm start` and open a web app url (`http://localhost:1234`)
4) Make sure the page is visible 

**Next plans**
If you accept the PR, I'll install and configure `TypeScript` and `TSLint`
